### PR TITLE
Added PDP_DATA_UPDATE_CALLBACKS config

### PR DESCRIPTION
**Pull request by danyi1212**

### CODE REVIEW

File: docs/concepts/pdp/configuration.mdx [hunk 1]
```diff
@@ -264,6 +264,31 @@ Default: `<green>{time}</green> | {process} | <blue>{name: <40}</blue>|<level>{l
...
+      "data": {
+        "key": "value"
+      }
+    }
+  }
+]
+```
+The default config is: `{"method": "POST"}`.
+
+For more information, see the [CallbackEntry](https://github.com/permitio/opal/blob/master/packages/opal-common/opal_common/schemas/data.py#L123) interface.
```
Comment (omer9564) [inline docs/concepts/pdp/configuration.mdx:291]: missing danger that having slow API receiving the callbacks might affect the PDP synchronization time

**Review by omer9564 — CHANGES_REQUESTED** [review]

File: docs/concepts/pdp/configuration.mdx [hunk 2]
```diff
@@ -264,6 +264,31 @@ Default: `<green>{time}</green> | {process} | <blue>{name: <40}</blue>|<level>{l
...
+
+Default: `[]`
+
+List of data update callbacks to be called when data is updated. For example:
+```json
+[
+  {
+    "url": "http://example.com/callback",
+    "key": "example",  // optional
+    "config": {  // optional
```
Comment (omer9564) [inline docs/concepts/pdp/configuration.mdx:277]: this isn't a valid json

**Review by omer9564 — APPROVED** [review]
approved with comments

File: docs/concepts/pdp/configuration.mdx (hunk 2 above)
Comment (danyi1212) [reply docs/concepts/pdp/configuration.mdx:277]: Right, it's docs it does not have to be valid 🙂

**Pull request merged**